packages/core/admin/admin/src/hooks/index.js

```diff
@@ -3,7 +3,6 @@ export { useContentTypes } from './useContentTypes';
 export { default as useFetchPermissionsLayout } from './useFetchPermissionsLayout';
 export { default as useFetchRole } from './useFetchRole';
 export { default as useLicenseLimitNotification } from './useLicenseLimitNotification';
-export { default as useLicenseLimit } from './useLicenseLimits';
```

Comment:
This could have been done on `main` but I missed it. The name of the export is wrong and the hook isn't used anywhere in CE.